realm/realm-library/src/syncIntegrationTest/java/io/realm/SSLConfigurationTests.java

```diff
@@ -42,7 +42,7 @@
 public class SSLConfigurationTests extends StandardIntegrationTest {
 
     @Rule
-    public Timeout globalTimeout = Timeout.seconds(10);
+    public Timeout globalTimeout = Timeout.seconds(120);
```

Comment:
really need 120 seconds long?

Reply:
This is a consequence of increasing the timeout in [HttpUtil](https://github.com/realm/realm-java/pull/5511/files#diff-2365fb22852452691d8d2c0019e0ab5bR39) since I'm experiencing a longer ROS time startup see https://github.com/realm/realm-object-server-private/issues/697